packages/sidenav/src/Sidenav.ts

```diff
@@ -74,6 +74,9 @@ export class SideNav extends Focusable {
     @property({ reflect: true })
     public value: string | undefined = undefined;
 
+    @property({ reflect: true })
```

Comment:
Can you add a description as to what this does so that it populates to: https://63ed2ff266722307437d351a--spectrum-web-components.netlify.app/components/sidenav/api/#attributes-and-properties

<img width="996" alt="image" src="https://user-images.githubusercontent.com/1156657/219193664-254ec4b5-2e86-45d8-b8dd-b57c864e99ae.png">

Local preview available via `yarn docs:start`. 

Here's an example of a property with a description: https://github.com/adobe/spectrum-web-components/blob/main/packages/accordion/src/Accordion.ts#L42

As seen in: https://63ed2ff266722307437d351a--spectrum-web-components.netlify.app/components/accordion/api/

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/1156657/219194594-092c4d3d-a487-4ed4-9267-eb202ddc57fd.png">

Reply:
Done.
<img width="999" alt="image" src="https://user-images.githubusercontent.com/1207520/219215834-d071e62a-958b-49ce-969f-4c1770f71d31.png">

Reply:
I also set the default value as `undefined`.

Reply:
Interesting, does it not set a default as `undefined` by way of the `?` usage? TIL!

Thanks for this.

Reply:
The default column will be empty if I don't explicitly set it as undefined.